### DISCUSSION TITLE
early days

### DISCUSSION THREAD

**Discussion by gernest** [vinceanalytics/vince#12] [General]
Yesterday I took a selfie on my workstation. I had the same feeling I felt while reading about early days of computing. Just ordinary people dreaming of doing extraordinary things, one bit at a time.

I hope vince experiment will pan out and disrupt analytics landscape in my Country.

![20240827_183731](https://github.com/user-attachments/assets/4bb64774-b484-41ff-ba12-eec8d7e8a4d4)

**Comment by Martin-Fox** [comment]
I have one question - what ports are used by Vince? Only 433 or something else?

**Reply by gernest** [reply]
Any port, use `--liisten=:8080`  flag to listen to port `8080` or change it to any number

**Reply by Martin-Fox** [reply]
Ok, I found the reason why it was not counting visitors. Wrong Cloudflare tunnel configuration.

**Reply by gernest** [reply]
nice, glad your issue was resolved!

**Reply by DasCanard** [reply]
> Ok, I found the reason why it was not counting visitors. Wrong Cloudflare tunnel configuration.

hey, how did you fix it?
